pol-core/plib/uofile00.cpp

```diff
@@ -43,10 +119,21 @@ bool open_uopmap_file( const int mapid )
   }
 
   uopmapfile.open( filename, std::ios::binary );
-  return (bool)uopmapfile;
+
+  if ( (bool)uopmapfile )
```

Comment:
if (uopmapfile.good()) or !uopmapfile.fail() would be clearer to read, so you don't have to check the type of the variable to check if the standard defines a operator bool ;)